app/extensions/brave/locales/pt-BR/app.properties

```diff
@@ -78,11 +78,11 @@ blockedTrackingElements={{blockedTrackingElementsSize}} Elementos de rastreament
 replacedAds={{replacedAdsSize}} Anúncios substituídos
 blockedAds={{blockedAdsSize}} Anúncios bloqueados
 redirectedResources={{redirectedResourcesSize}} Recursos atualizados para HTTPS
-updateNow=atualizar agora
+updateNow=Atualizar agora
 updateChecking=Estamos verificando atualizações...
 updateDownloading=Atualização encontrada, estamos baixando...
-updateLater=mais tarde
-updateDetails=detalhes
+updateLater=Mais tarde
+updateDetails=Detalhes
```

Comment:
You can request access here: https://www.transifex.com/brave/brave-laptop/dashboard/